src/coreComponents/constitutive/solid/Damage.hpp

```diff
@@ -116,11 +127,11 @@ class DamageUpdates : public UPDATE_BASE
 
     if( m_extDrivingForceFlag )
     {
-      pf = fmax( fmin( 1.0, m_damage( k, q )), 0.0 );
+      pf = fmax( fmin( 1.0, m_newDamage( k, q )), 0.0 );
```

Comment:
I don't think those std functions are portable. should use:
```suggestion
      pf = LvArray::math::max( LvArray::math::min( 1.0, m_newDamage( k, q )), 0.0 );
```